docs/csharp/language-reference/keywords/explicit-numeric-conversions-table.md

```diff
@@ -10,52 +10,52 @@ helpviewer_keywords:
+# Explicit numeric conversions table (C# Reference)
+
+The following table shows the predefined explicit conversions between .NET numeric types, for which there is no [implicit conversion](implicit-numeric-conversions-table.md).
+
 |From|To|  
 |----------|--------|  
-|[sbyte](../../../csharp/language-reference/keywords/sbyte.md)|`byte`, `ushort`, `uint`, `ulong`, or `char`|  
-|[byte](../../../csharp/language-reference/keywords/byte.md)|`Sbyte` or `char`|  
-|[short](../../../csharp/language-reference/keywords/short.md)|`sbyte`, `byte`, `ushort`, `uint`, `ulong`, or `char`|  
-|[ushort](../../../csharp/language-reference/keywords/ushort.md)|`sbyte`, `byte`, `short`, or `char`|  
-|[int](../../../csharp/language-reference/keywords/int.md)|`sbyte`, `byte`, `short`, `ushort`, `uint`, `ulong`,or `char`|  
-|[uint](../../../csharp/language-reference/keywords/uint.md)|`sbyte`, `byte`, `short`, `ushort`, `int`, or `char`|  
-|[long](../../../csharp/language-reference/keywords/long.md)|`sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `ulong`, or `char`|  
-|[ulong](../../../csharp/language-reference/keywords/ulong.md)|`sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `long`, or `char`|  
-|[char](../../../csharp/language-reference/keywords/char.md)|`sbyte`, `byte`, or `short`|  
-|[float](../../../csharp/language-reference/keywords/float.md)|`sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `long`, `ulong`, `char`,or `decimal`|  
-|[double](../../../csharp/language-reference/keywords/double.md)|`sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `long`, `ulong`, `char`, `float`,or `decimal`|  
-|[decimal](../../../csharp/language-reference/keywords/decimal.md)|`sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `long`, `ulong`, `char`, `float`, or `double`|  
+|[sbyte](sbyte.md)|`byte`, `ushort`, `uint`, `ulong`, or `char`|  
+|[byte](byte.md)|`sbyte` or `char`|  
+|[short](short.md)|`sbyte`, `byte`, `ushort`, `uint`, `ulong`, or `char`|  
+|[ushort](ushort.md)|`sbyte`, `byte`, `short`, or `char`|  
+|[int](int.md)|`sbyte`, `byte`, `short`, `ushort`, `uint`, `ulong`,or `char`|  
+|[uint](uint.md)|`sbyte`, `byte`, `short`, `ushort`, `int`, or `char`|  
+|[long](long.md)|`sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `ulong`, or `char`|  
+|[ulong](ulong.md)|`sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `long`, or `char`|  
+|[char](char.md)|`sbyte`, `byte`, or `short`|  
+|[float](float.md)|`sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `long`, `ulong`, `char`,or `decimal`|  
+|[double](double.md)|`sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `long`, `ulong`, `char`, `float`,or `decimal`|  
+|[decimal](decimal.md)|`sbyte`, `byte`, `short`, `ushort`, `int`, `uint`, `long`, `ulong`, `char`, `float`, or `double`|  
   
 ## Remarks  
   
--   The explicit numeric conversion may cause loss of precision or result in throwing exceptions.  
+- The explicit numeric conversion may cause loss of precision or result in throwing exceptions.  
```

Comment:
in throwing exceptions --> in throwing an exception, typically an \<xref:System.OverflowException>.